backend/src/main/java/gov/cdc/usds/simplereport/api/converter/FhirConverter.java

```diff
@@ -582,6 +588,9 @@ public Observation convertToObservation(ConvertToObservationProps props) {
         .addInterpretation()
         .addCoding(convertToAbnormalFlagInterpretation(props.getResultCode()));
 
+    observation.setIssued(props.getIssued());
+    observation.getIssuedElement().setTimeZone(TimeZone.getTimeZone("utc"));
```

Comment:
setting the time zone does not change the value of the date itself, but enforces the time zone in the message to be consistent. `observation.setIssued()` on its own converts the Date parameter into the system local time zone so while the value is accurate it won't be consistently displayed in the same time zone in local testing

Reply:
Should we always use the `+00:00` datetime format instead of `Z`? I was searching for guidance on it with #5985 since in that I had used `setTimeZoneZulu(true)` which outputs it with `Z` instead of `setTimeZone(TimeZone.getTimeZone("utc"))` which outputs `+00:00`

Reply:
I don't have a preference either way - I think the important thing is just that we have consistent output. since you're doing the bulk of the work if you prefer the format with Z I will change that here - it does look a little cleaner in the code that way imo

Reply:
actually I want to amend this - is there a way to get DiagnosticReport.effective to be in the format with Z instead of with +0:00?
else it's easier to match the format for that field. checking...